ui/appui-react/src/appui-react/ui-items-provider/UiItemsManager.ts

```diff
@@ -126,6 +127,7 @@ export class UiItemsManager {
   /**
    * Registers a UiItemsProvider with the UiItemsManager.
    * @param uiProvider the UI items provider to register.
+   * @param overrides the UI items provider to register.
    */
   public static register(
```

Comment:
One idea to avoid this issue is to avoid global/static register functions in packages/modules and instead export a provider itself (which can later be overridden by an application when registering).